root/static/scripts/guess-case/MB/Control/GuessCase.js

```diff
@@ -60,7 +61,7 @@ var mode = ko.computed({
 
         if (modeName !== gc.modeName) {
             gc.modeName = modeName;
-            gc.mode = MB.GuessCase.Mode[modeName];
+            gc.mode = require('../../modes')[modeName];
```

Comment:
http://babeljs.io/docs/learn-es2015/ is pretty good I think. But it helps to understand [Node.js's module system](https://nodejs.org/api/modules.html) too, since that's what Babel converts the ES6 `import` statements into.

The `require` function always returns module's `exports` object. Statements like `export let English = ...` are converted into something like `exports.English = ...`.  So `modeName` would be English, French, Sentence, etc.

This should probably be moved to an `import` at the top of the file, anyway.